solr/CHANGES.txt

```diff
@@ -8,6 +8,8 @@ https://github.com/apache/solr/blob/main/solr/solr-ref-guide/modules/upgrade-not
 
 New Features
 ---------------------
+* SOLR-16466: Admin UI - Make it optional to sort list of commandline args (Shawn Heisey, Vincenzo D'Amore via Christine Poerschke)
```

Comment:
```suggestion
* SOLR-16466: Admin UI - Make it optional to sort list of commandline args (Shawn Heisey, Vincenzo D'Amore)
```

I didn't do much here really.

Reply:
I don't agree you pointed me straight to the line where the problem was. All I had to do was write a more flexible implementation and test it.